src/namespaces/hardhat.rs

```diff
@@ -131,4 +131,16 @@ pub trait HardhatNamespaceT {
     /// A `BoxFuture` containing a `Result` with a `bool` representing the success of the operation.
     #[rpc(name = "hardhat_setStorageAt")]
     fn set_storage_at(&self, address: Address, slot: U256, value: U256) -> RpcResult<bool>;
+
+    /// Directly modifies the L1 gas price.
```

Comment:
L1 gas price>

Reply:
Yes, as stored in the node's fee model - do you want another comment?